Can't initialize the build of the repo

Doing `pnpm install` shows the following:
```shell
[divya@archdivya l2-settlements-contracts]$ pnpm install
Lockfile is up to date, resolution step is skipped
Packages: +1059
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/@t3rn/pricer/-/pricer-1.7.9.tgz: Not Found - 404

No authorization header was set for the request.
Progress: resolved 1059, reused 1058, downloaded 0, added 0
```

--- a/package.json
+++ b/package.json
@@ -54,7 +54,6 @@
     "@polkadot/api": "^10.13.1",
     "@sentry/node": "^7.111.0",
     "@sentry/profiling-node": "^7.111.0",
-    "@t3rn/pricer": "1.7.9",
     "axios": "^1.6.8",
     "cors": "^2.8.5",
     "crypto-convert": "^2.1.7",

